kyuubi-server/src/test/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilderSuite.scala

```diff
@@ -26,7 +26,10 @@ class FlinkProcessBuilderSuite extends KyuubiFunSuite {
   test("flink engine process builder") {
     val builder = new FlinkProcessBuilder("vinoyang", conf)
     val commands = builder.toString.split(' ')
-    assert(commands.exists(_ endsWith "flink-sql-engine.sh"))
+    assert(commands.head.endsWith("bin/java"), "wrong exec")
```

Comment:
Why not directly assert the `commands.mkString(" ")`?

Reply:
> Why not assert the `commands.mkString(" ")`?
commands contain full paths to many jars. Concatenating these strings in UT is complicated. So it is reasonable to keep the original UT

Reply:
> Why not assert the `commands.mkString(" ")`?

OK